console-extensions.json

```diff
@@ -63,20 +63,13 @@
     }
   },
   {
-    "type": "console.resource/create",
+    "type": "console.page/route",
     "properties": {
-      "name": "default",
-      "model": {
-        "group": "stable.example.com",
-        "kind": "CronTab",
-        "version": "v1"
-      },
+      "path": "/k8s/ns/:ns/stable.example.com~v1~CronTab/~new/form",
       "component": {
         "$codeRef": "CronTabForm"
       },
-      "flags": {
-        "required": ["CRONTAB"]
-      }
+      "exact": true
```

Comment:
This is where it's good to have the side-by-side comparison!  

This addition should really be protected by a flag.  But note https://github.com/openshift/console-crontab-plugin/pull/35.  The existing flags are in the wrong location, so they don't work.  So let's merge this as is and I will add the flag in https://github.com/openshift/console-crontab-plugin/pull/35.